airflow/providers/amazon/aws/sensors/emr.py

```diff
@@ -111,6 +116,152 @@ def failure_message_from_response(response: Dict[str, Any]) -> Optional[str]:
         raise NotImplementedError('Please implement failure_message_from_response() in subclass')
 
 
+class EmrServerlessJobSensor(BaseSensorOperator):
+    """
+    Asks for the state of the job run until it reaches a failure state or success state.
+    If the job run fails, the task will fail.
+
+    .. seealso::
+        For more information on how to use this sensor, take a look at the guide:
+        :ref:`howto/sensor:EmrServerlessJobSensor`
+
+    :param application_id: application_id to check the state of
+    :param job_run_id: job_run_id to check the state of
+    :param target_states: a set of states to wait for, defaults to SUCCESS_STATES
```

Comment:
```suggestion
    :param target_states: a set of states to wait for, defaults to 'SUCCESS'
```
Maybe it makes sense to use the actual value being checked rather than the variable in the docstring. I was thinking users reading the Python API docs won't have to look at code or other places in the docs to figure out what `SUCCESS_STATES` is. WDYT?

Reply:
Agree